open-craft/openedx-completion-aggregator
# [BB-2271] Fix cohort updated signal handler

**Description:** User fails serialization when not in eager mode. This PR passes usernames instead to the background task as that's what's really needed.

**JIRA:** BB-2271

**Merge deadline:** ASAP

**Testing instructions:**

1. Deploy to integration and ensure operations that update a user cohort does not crashes.

**Reviewers:**

**Merge checklist:**

**Post merge:**
      finished.
